project/requirements.md

```diff
@@ -0,0 +1,20 @@
+### Final Project: Calculator
+
+#### Due Date
+A pull request with your code must be issued to the unit-1 `calculator` branch by 18:59 on 5/12 (that is, before class). Presentations will be in class that day.
+
+#### Groups
+You will work in assigned [groups of two](groups.md).
+
+#### Description
+
+Create a calculator Android application. This application should be both well-designed and functional. Feel free to add additional features beyond the requirements of the application. When submitting your project, include a breakdown of what each group member did (and be prepared to talk about this during the demo) and include a README.md which explains the overall structure of your code.
+
+#### Requirements
+* Handle user input and display the results.
```

Comment:
Good note. And maybe add a list of "scientific" features, and link to the [Google search scientific calc](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&es_th=1&ie=UTF-8#q=scientific%20calculator)?